streams/src/test/java/org/apache/kafka/streams/integration/PurgeRepartitionTopicIntegrationTest.java

```diff
@@ -157,6 +158,7 @@ public void setup() {
         streamsConfiguration.put(StreamsConfig.STATE_DIR_CONFIG, TestUtils.tempDirectory(APPLICATION_ID).getPath());
         streamsConfiguration.put(StreamsConfig.topicPrefix(TopicConfig.SEGMENT_MS_CONFIG), purgeIntervalMs);
         streamsConfiguration.put(StreamsConfig.topicPrefix(TopicConfig.SEGMENT_BYTES_CONFIG), purgeSegmentBytes);
+        streamsConfiguration.put(StreamsConfig.topicPrefix(TopicConfig.RETENTION_MS_CONFIG), purgeRetention);
```

Comment:
Do we need this setting here? I seem to duplicate 
```
tempTopicDefaultOverrides.put(TopicConfig.RETENTION_MS_CONFIG, String.valueOf(Long.MAX_VALUE));
```
from `RepartitionTopicConfig.java` ?